### ISSUE TITLE
Duplicate file open errors

### ISSUE THREAD

**Issue by PropGit** [dbetz/proploader#21]
Proploader v1.0-11

Trying to open a non-existent file leads to two 103 messages.

```
C:\>proploader -c -p COM1 Welcome.elfs
001-Opening file 'Welcome.elfs'
103-ERROR: Can't open file 'Welcome.elfs'
103-ERROR: Can't open file 'Welcome.elfs'
```

**Comment by PropGit** [comment]
Tested with v1.0-18.  This issue still occurs.

**Comment by dbetz** [comment]
Ummm... I never looked into this one. I'll check now.

**Comment by dbetz** [comment]
I pushed a fix for this. It should appear in the next build.

**Comment by PropGit** [comment]
Verified that it's fixed in v1.0-22.
